docs/modules/mongodb_watchdog.md

```diff
@@ -1 +1,15 @@
 # Logger: `mongodb_watchdog`
+
+The module uses a separate database, using the logger database alias in settings. Do NOT point that alias to the same database as default, because the module drops the logger database when uninstalling, which would drop all your other data with it.
+
+* mongodb.watchdog.items: the maximum item limit on the capped collection used by the module. If not defined, it defaults to 10000. The actual (size-based) limit is derived from this variable, assuming 1 kiB per watchdog entry.
+
```

Comment:
I think it is better to add subitems for details about the variables rather than lump everything together in one paragraph. What's your opinion on this ? Also, we probably need to add a link to the kiB definition in wikipedia because some users may not be familiar with binary units.